.github/workflows/go.yml

```diff
@@ -0,0 +1,155 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build-and-test:
+    name: Build Go and Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+        # Needed for changelog generation by goreleaser
+      - name: Unshallow
+        run: git fetch --prune --unshallow
+
+      - name: Get dependencies
+        run: make deps
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Test (Linux/macOS)
+        if: matrix.os != 'windows-latest'
+        run: make coverage
+        timeout-minutes: 3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SYSL_PLANTUML: http://www.plantuml.com/plantuml
+
+      - name: Test (Windows)
+        if: matrix.os == 'windows-latest'
+        run: .\scripts\test-with-coverage-windows.bat
+        timeout-minutes: 3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SYSL_PLANTUML: http://www.plantuml.com/plantuml
+
+      - name: Set GOVERSION environment variable (Linux/macOS)
+        if: matrix.os != 'windows-latest'
+        run: echo ::set-env name=GOVERSION::$(go version | awk '{print $3, $4;}')
+
+      - name: Set GOVERSION environment variable (Windows)
+        if: matrix.os == 'windows-latest'
+        run: $Env:GOVERSION=(go version | %{($_ -split " ")[-2,-1] -join " "}); echo $Env:GOVERSION
+
+      - name: Build sysl binary via goreleaser (Linux)
+        if: matrix.os != 'windows-latest'
```

Comment:
This step says "(Linux)" but the matrix test says "not windows". I think this is meant to be `== ubuntu-latest`?